application/src/main/resources/application.properties

```diff
@@ -121,5 +121,6 @@ flyway.noop.strategy=${FLYWAY_NOOP_STRATEGY:false}
 spring.flyway.baseline-on-migrate=true
 spring.flyway.baseline-version=0001
 spring.flyway.out-of-order=true
+spring.flyway.ignore-missing-migrations=true
```

Comment:
you might need to update gradle config as well.

Reply:
added